src/app/features/home/components/news/news-list/news-list.component.ts

```diff
@@ -10,12 +10,12 @@ import {NewsService} from "../../../services/news.service";
 })
 export class NewsListComponent implements OnInit {
 
-  cards: Observable<Array<Item>> | undefined;
+  newsCards: Observable<Array<NewsItem>> | undefined;
```

Comment:
Please turn off eslint rule for the case that all vars should have an initial value